examples/developer-tools/src/storage.ts

```diff
@@ -113,3 +115,22 @@ export const getAllSavedBlockNames = function (): Set<string> {
 export const getProhibitedBlockNames = function (): Set<string> {
   return prohibitedBlockNames;
 };
+
+export interface BlockFactorySettings {
+  blockDefinitionFormat: string;
+  codeHeaderStyle: string;
+  codeGeneratorLanguage: string;
+}
+
+export const saveBlockFactorySettings = function (
```

Comment:
```suggestion
export function saveBlockFactorySettings (
```
https://google.github.io/styleguide/tsguide.html#function-declarations

Reply:
Fixed em all here and elsewhere.  Thanks!